Rakefile

```diff
@@ -1,2 +1,88 @@
+# frozen_string_literal: true
```

Comment:
I don't understand how all these Rakefile changes relate to adding Sensitive type